nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-services-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,327 @@
+nifi-cassandra-services-nar

```

Comment:
Missing logback dependency reference

Reply:
Missing notice for `io.dropwizard.metrics:metrics-core:jar:3.2.2:compile`

Reply:
Missing notice for groupId `org.ow2.asm`

Reply:
Missing notice for Netty

Reply:
Missing notice for `de.svenkubiak:jBCrypt`

Reply:
Missing notices for this entire branch:

```
[INFO] |     |  +- org.antlr:antlr-runtime:jar:3.5.2:compile
[INFO] |     |  +- com.jayway.jsonpath:json-path:jar:2.4.0:compile
[INFO] |     |  |  \- net.minidev:json-smart:jar:2.3:compile
[INFO] |     |  |     \- net.minidev:accessors-smart:jar:1.2:compile
```

Reply:
Fixed all. Please take a look.